cmake/modules/FindHostTools.cmake

```diff
@@ -116,4 +116,4 @@ set(HostTools_FOUND TRUE)
 set(HOSTTOOLS_FOUND TRUE)
 build_info(toolchain name VALUE ${ZEPHYR_TOOLCHAIN_VARIANT})
 string(TOUPPER ${ZEPHYR_TOOLCHAIN_VARIANT} zephyr_toolchain_variant_upper)
-build_info(toolchain path VALUE "${${zephyr_toolchain_variant_upper}_TOOLCHAIN_PATH}")
+build_info(toolchain path DIR VALUE "${${zephyr_toolchain_variant_upper}_TOOLCHAIN_PATH}")
```

Comment:
`DIR` is odd to use for paths, why not `PATH`?

Reply:
Maybe even simpler to have either `VALUE` or `PATH`?

Reply:
> Maybe even simpler to have either VALUE or PATH?

I did consider that, but the reason I decided to not do so was because paths which are constructed from internal CMake variables, like CMAKE_SOURCE_DIR and friends, then those paths are guaranteed to be CMake style.
Thus no conversion of `\` is needed and values can be used directly and we can apply `DIR`(or `PATH`) only on path provided by user.

But agreed, from developer perspective it probably makes more sense to just use `PATH` and not worry about this small penalty on a non-conversion.

Reply:
Done